tools/gyp/pylib/gyp/generator/msvs.py

```diff
@@ -2662,6 +2662,15 @@ def _GetMSBuildGlobalProperties(spec, guid, gyp_file_name):
     else:
       properties[0].append(['ApplicationType', 'Windows Store'])
 
+  msvs_windows_sdk_version = None
+  if msvs_windows_sdk_version == None and version.ShortName() == '2017':
+    vs2017_sdk = '10.0.14393.0'
```

Comment:
I would rather see an early fail then use an arbitrary default.
P.S. When "Windows Creators Edition" will be released the default version will be `10.0.15052.*`